cmake-proxies/cmake-modules/DependenciesList.cmake

```diff
@@ -0,0 +1,83 @@
+# conan_package_options variable can be used to pass additional options to the conan install command.
+
+audacity_find_package(ZLIB REQUIRED)
+audacity_find_package(EXPAT REQUIRED)
+audacity_find_package(PNG QUIET CONAN_PACKAGE_NAME libpng)
+audacity_find_package(JPEG QUIET CONAN_PACKAGE_NAME libjpeg-turbo)
+
+audacity_find_package(wxWidgets REQUIRED FIND_PACKAGE_OPTIONS COMPONENTS adv base core html qa xml net)
+
+audacity_find_package(libmp3lame REQUIRED)
+
+audacity_find_package(mpg123 OPTION_NAME libmpg123)
+
+if( NOT ${_OPT}use_libmpg123 STREQUAL "off" )
+   # If we are building against libmpg123, we need to drop
+   # the previos configuration, which may used libmad
+   set( USE_LIBMAD OFF CACHE INTERNAL "" FORCE )
+   set( ${_OPT}use_libmad "off" )
+else()
+   audacity_find_package(libmad)
+endif()
+
+audacity_find_package(libid3tag)
+
+audacity_find_package(WavPack)
+audacity_find_package(Ogg OPTION_NAME libogg)
+audacity_find_package(FLAC OPTION_NAME libflac)
+audacity_find_package(Opus OPTION_NAME libopus)
+audacity_find_package(Vorbis OPTION_NAME libvorbis)
+audacity_find_package(SndFile CONAN_PACKAGE_NAME libsndfile OPTION_NAME libsndfile)
+
+# Allow building with ASIO for windows users
+cmake_dependent_option(
+   ${_OPT}has_asio_support
+   "Build Audacity with ASIO support"
+   False
+   "WIN32 AND IS_64BIT"
+   False
+)
+
+# Conan expects Python boolean, CMake will convert values to
+# ON/OFF
+
+if( ${_OPT}has_asio_support )
+   list(APPEND conan_package_options "use_asio=True")
+endif()
+
+if(UNIX AND NOT APPLE)
+   find_package(JACK QUIET)
```

Comment:
Is there a reason JACK is handled differently here than the other packages (e.g. using standard `find_package` instead of the new `audacity_find_package`)?

Reply:
Yes, we never build Jack. This keeps the old behavior actually.